src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -844,11 +849,7 @@ export default class BoxScatterPlot<
                     role="img"
                     viewBox={`0 0 ${this.svgWidth} ${this.svgHeight}`}
                     onMouseMove={this.onMouseMove}
-                    ref={ref => {
-                        if (this.props.svgRef) {
-                            this.props.svgRef(ref);
-                        }
-                    }}
+                    ref={this.svgRef()}
```

Comment:
it's kind of strange that this gets invoked.  usually you just pass a callback to ref

Reply:
like, why don't you just feed it this.props.svgRef ?